src/SFML/Graphics/Image.cpp

```diff
@@ -131,6 +133,14 @@ sfBool sfImage_saveToFile(const sfImage* image, const char* filename)
 }
 
 
+////////////////////////////////////////////////////////////
+sfBool sfImage_saveToMemory(const sfImage* image, sfBuffer* output, const char* format)
+{
+    CSFML_CHECK_RETURN(output, sfFalse);
+    CSFML_CALL_RETURN(image, saveToMemory(output->vector, format), sfFalse);
```

Comment:
`format` gets converted to `std::string` and it's UB to construct a string from a null pointer. Should we add a `CSFML_CHECK_RETURN` call for `format` being null?

Reply:
Nice catch, will do.

Reply:
That's fine by me